src/js/gi/selectors/calculator.js

```diff
@@ -183,15 +177,18 @@ const getDerivedValues = createSelector(
     } else if (isCorrespondence) {
       tuitionFeesCap = constant.CORRESPONDTFCAP;
     } else if (isPublic && institution.country === 'usa') {
-      tuitionFeesCap = inputs.inState
+      tuitionFeesCap = inputs.inState === 'yes'
                      ? institution.tuitionInState
                      : institution.tuitionOutOfState;
     } else if (isPrivate || isForeign) {
       tuitionFeesCap = constant.TFCAP;
+    } else {
+      // TODO: What should default cap be?
```

Comment:
TFCAP is one of the most important constants in the tool... it is the private school tuition cap for the Post-9/11 GI Bill.

TFCAP =21970.46

It can be found in the current version of the tool in two places:

**Estimator (LINE 31)**
https://github.com/department-of-veterans-affairs/gi-bill-comparison-tool/blob/master/app/assets/js/estimator.js

![image](https://cloud.githubusercontent.com/assets/4397508/24095640/69b7a118-0d34-11e7-89a2-2a6e7cf17207.png)

**Calculator (LINE 54)**
https://github.com/department-of-veterans-affairs/gi-bill-comparison-tool/blob/production/app/assets/js/calculator.js

![image](https://cloud.githubusercontent.com/assets/4397508/24095665/88d67272-0d34-11e7-9709-1d0017befbb1.png)